src/main/java/roomescape/ReservationApiController.java

```diff
@@ -0,0 +1,63 @@
+package roomescape;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.concurrent.atomic.AtomicLong;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("reservations")
```

Comment:
```suggestion
@RequestMapping("/reservations")
```

`/`를 생략하기 보다 포함시켜주는 편입니다.

Reply:
감사합니다. /를 포함시켜 해당 문자열이 url임을 나타낼 수 있으면 가독성이 좋아지겠군요 😊